### ISSUE TITLE
Custom Tables Functional Testing Needed

### ISSUE THREAD

**Issue by spficklin** [tripal/tripal#1345]
Support for custom tables has been added with PR tripal/t4d8#209 . However, we don't yet have the functional testing required as part of that PR. we cannot perform the function tests until issue tripal/t4d8#217 is resolved because it will cause errors.  This issue is to remind us to do that functional testing.

**Comment by Ferrisx4** [comment]
Functional testing for Custom Tables was included in https://github.com/tripal/tripal/pull/1771. I'm going to close this issue.

**Comment by Ferrisx4** [comment]
These tests were implemented as a service in #1771, which renders [the original test file](https://github.com/tripal/tripal/blob/79a2a626faaea99b124061139aaab9d30a88662e/tripal_chado/tests/src/Functional/ChadoCustomTables/ChadoCustomTableTest.php) probably unnecessary. I'm going to delete it if there's no objections.

**Comment by laceysanderson** [comment]
Good catch, they were duplicated. I agree with removing the original test in Functional/CustomTables in favour of the new test in Functional/Services 👍
